README.fr.md

```diff
@@ -5,7 +5,7 @@
 
 # FreshRSS
 
-FreshRSS est un agrégateur de flux RSS à auto-héberger à l’image de [Leed](https://github.com/LeedRSS/Leed) ou de [Kriss Feed](https://tontof.net/kriss/feed/).
```

Comment:
The link https://tontof.net/kriss/feed/ was dead

Reply:
The project is still available https://github.com/tontof/kriss_feed.
We might just change the link target.

Reply:
It is mostly that I do not find it relevant (anymore), at least not in an intro. My reasoning is that this paragraph is to explain what FreshRSS is, and the likelihood of newcomers to be enlightened by naming Leed or Kriss is probably near zero.
If we are to give an example, naming Feedly (or even Google Reader) might be more helpful.
If relevant, we could have a new section further down or in the documentation with some selected alternatives.